app/pages/events/event_list/event_list.php

```diff
@@ -44,8 +44,19 @@
 
 <?php if ($can_edit): ?>
     <nav id="page-actions">
-        <a href="/evenements/nouveau"><i class="fas fa-plus"></i> Ajouter un
-            événement</a>
+        <li>
+            <details class="dropdown">
+                <summary>Actions</summary>
+                <ul>
+                    <li><a href="/evenements/nouveau" class="secondary">
+                            <i class="fas fa-plus"></i> Ajouter un événement
+                        </a></li>
+                    <!-- <li><a href="/ajouter-activite" class="secondary">
+                            <i class="fas fa-plus"></i> Ajouter une activité
+                        </a></li> -->
```

Comment:
Can you rename routes to be more consistent: `ajouter-evenement` and `ajouter-activite`

Reply:
I renamed the `ajouter-activite` into `activite/nouveau`, seemed more consistent with the logic of the routes